packages/components/src/components/bal-slider/bal-slider.tsx

```diff
@@ -8,6 +8,11 @@ import { observeItems } from '../../utils/observer'
 export class Slider implements ComponentInterface {
   @Element() el!: HTMLBalSliderElement
 
+  /**
+   * The list of names for pagination tabs
+   */
+  @Prop() tabs: string[] = []
```

Comment:
- [ ] provide the labels of the slider items with the slider items element

because then they are deeply connected and later it is much easier to adde other stuff to the items like an icon

Reply:
so please add a prop called label to the pal-slider-item